Essay_Analysis_Tool/Notepad-Sharp.csproj

```diff
@@ -58,8 +63,10 @@
     <Compile Include="Business\DeclarationSnippet.cs" />
     <Compile Include="Business\InsertEnterSnippet.cs" />
     <Compile Include="Business\InsertSpaceSnippet.cs" />
-    <Compile Include="Business\LoggerViewController.cs" />
-    <Compile Include="Business\MainViewController.cs" />
+    <Compile Include="Business\LoggingService.cs" />
+    <Compile Include="DAL\DBEngine.cs" />
+    <Compile Include="Presentation\LoggerViewController.cs" />
+    <Compile Include="Presentation\MainViewController.cs" />
```

Comment:
Controllers are not part of the presentation layer.
src/
  |-> Controllers
  |-> Presentation
  |-> Services

Reply:
I did a little research and discovered that controllers are not part of the business logic at all since anytime users interact with the UI, they are actually supposed to be interacting with the controllers and modifying data in the data access layer, which I am building at the moment, but yes, I do like the idea of separating those out into their own 'Controllers' folder.